rswag-specs/spec/rswag/specs/example_group_helpers_spec.rb

```diff
@@ -159,13 +171,13 @@ module Specs
           let(:example_two_name) { 'example_two' }
           let(:example_two_summary) { 'An example description' }
 
-          it "assigns all examples to the metadata" do 
-            expect(api_metadata[:operation][:request_examples].length()).to eq(2) 
+          it "assigns all examples to the metadata" do
+            expect(api_metadata[:operation][:request_examples].length()).to eq(2)
             expect(api_metadata[:operation][:request_examples][0]).to eq({ value: example_one, name: 0 })
             expect(api_metadata[:operation][:request_examples][1]).to eq({ value: example_two, name: example_two_name, summary: example_two_summary })
-          end 
-        end  
-      end 
```

Comment:
VSCode automatically deleted extra whitespace on lines 139-168.  This change is unrelated to the PR.  Let me know if you'd prefer me to not include these changes.